server/data/incidentClient.ts

```diff
@@ -280,4 +282,36 @@ export default class IncidentClient {
       values: [formResponse, incidentDate, reportId],
     })
   }
+
+  async getReportInProgress(bookingId: number, status: string): Promise<inProgressReport> {
```

Comment:
Don't think this method name is accurate: `InProgress`.. its only `InProgress` if you pass in the `IN_PROGRESS` status.

Reply:
method already removed